src/pip/_internal/pep425tags.py

```diff
@@ -160,11 +192,14 @@ def get_platform():
         return 'macosx_{}_{}_{}'.format(split_ver[0], split_ver[1], machine)
 
     # XXX remove distutils dependency
-    result = distutils.util.get_platform().replace('.', '_').replace('-', '_')
-    if result == "linux_x86_64" and _is_running_32bit():
-        # 32 bit Python program (running on a 64 bit Linux): pip should only
-        # install and run 32 bit compiled extensions in that case.
-        result = "linux_i686"
+    elif sys.platform[:3] == 'aix':
+        result = _get_aix_pep425().replace('.', '_').replace('-', '_')
```

Comment:
I'd prefer it that _get_aix_pep425 return value doesn't need the replace calls here.

Since it's a function in pip's codebase and this is the only call site of that function, it'll be better to have it return the value we actually want to use.